includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -611,6 +562,76 @@ private function getFollowUs() {
 		return $data;
 	}
 
+	private function getCommunity() {
+		$data = [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-wikia-community-header'
+				]
+			],
+			'links' => [ ]
+		];
+
+		if ( $this->getHref( 'community-central' ) ) {
+			$data['links'][] = [
```

Comment:
Since all of these are exactly the same except for the link key, what do you think of just a loop over the keys? i.e.
```php
foreach ( $links as $link ) {
	$linkHref = $this->getHref( $link );
	if ( $linkHref ) {
		$data['links'][] = [
			'type' => 'link-text',
			'title' => [
				'type' => 'translatable-text',
				'key' => 'global-footer-wikia-community-link-' . $link,
			],
			'href' => $linkHref,
		];
	}
}
```